src/FEValues/cell_values.jl

```diff
@@ -97,6 +97,13 @@ function CellValues(::Type{T}, qr::QuadratureRule, ip::Interpolation, sgip::Scal
     return CellValues(T, qr, ip, VectorizedInterpolation(sgip))
 end
 
+################################################################# 
+# Note on dimensions:                                           #
+# sdim = spatial dimension (dimension of the grid)              # 
```

Comment:
```suggestion
# sdim = spatial dimension (dimension of the grid nodes)         # 
```

Reply:
We must have deleted them accidentally at some point. Sorry for the confusion. We also might want to add this to the devdocs or user docs to increase visibility.